docker-compose-dev.yml

```diff
@@ -0,0 +1,29 @@
+version: '3'
+
+volumes:
+  ez-dev-data:
+
+services:
+  backend:
+    image: ghcr.io/envelope-zero/backend:v1.0.0
+    volumes:
+      - ez-dev-data:/data
+    ports:
+      - '8080:8080'
+    environment:
+      API_URL: http://localhost:8080
+      CORS_ALLOW_ORIGINS: http://localhost:3000
+
+  frontend:
+    image: node:18.8.0-alpine
+    working_dir: /app
+    command: ['npm', 'run', 'start']
+    ports:
+      - '3000:3000'
+    volumes:
+      - ./:/app
+    environment:
+      REACT_APP_API_ENDPOINT: http://localhost:8080
+
```

Comment:
Is there any reason for that empty line here? :eyes:

Reply:
I think it looks weird having an empty line between the different env variables. But it doesn't really matter. Change it or leave it, up to you. :D